cedar-policy/src/tests.rs

```diff
@@ -2774,9 +2778,15 @@ mod schema_based_parsing_tests {
             }
         ]);
         let r = Entities::from_json_value(json, None).err().unwrap();
-        let expected_euid: cedar_policy_core::ast::EntityUID = r#"User::"alice""#.parse().unwrap();
         match r {
-            EntitiesError::Duplicate(euid) => assert_eq!(euid, expected_euid),
```

Comment:
We could update some of these tests to use `test_utils::expect_err` while were making changes